### PR TITLE
increase gradle timeouts to stabilize builds

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

We observed that some werft jobs fail sometimes with read timeout: https://gitpod.slack.com/archives/C01KGM9AW4W/p1643642672808109

This PR tries to improve the situation by increasing network timeouts as described here: https://stackoverflow.com/a/47607857/961588

## Related Issue(s)

Fixes #

## How to test

If build is green, merge and see whether it gets better next days.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **akosyakov** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-protocol/java/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/java/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**
- **[components/supervisor-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor-api/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mads-hartmann — APPROVED** [review]
Awesome, thanks!

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 5c46afe03cd7c78a9bfbb1663608f0fcd2c8bdac</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7941#pullrequestreview-868843096" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **akosyakov** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-protocol/java/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/java/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**
- **[components/supervisor-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor-api/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7941#issuecomment-1026621797" title="Approved">akosyakov</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7941#pullrequestreview-868843096" title="Approved">mads-hartmann</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7941#issuecomment-1026621797" title="Approved">akosyakov</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/gitpod-protocol/java/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/java/OWNERS)~~ [akosyakov]
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [akosyakov]
- ~~[components/supervisor-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor-api/OWNERS)~~ [akosyakov]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
